src/main/java/com/entropyteam/entropay/employees/dtos/EmployeeReportDto.java

```diff
@@ -71,8 +71,8 @@ public EmployeeReportDto(UUID id,
     public EmployeeReportDto(Employee employee, List<String> profile, Contract firstContract, Contract activeContract, String client, String project, List<String> technologiesName, Integer usdPayment, Integer arsPayment, String country, String labourEmail) {
```

Comment:
probably we should change the name of the param `activeContract` to `latestContract`